packages/node/src/evm/fulfillments/api-calls.ts

```diff
@@ -113,22 +118,39 @@ async function testAndSubmitFulfill(
   request: Request<ApiCall>,
   options: TransactionOptions
 ): Promise<LogsErrorData<SubmitResponse>> {
+  const errorMessage = requests.getErrorMessage(request);
+  if (errorMessage) {
+    const [submitLogs, submitErr, submitData] = await submitFail(airnodeRrp, request, errorMessage, options);
+    return [submitLogs, submitErr, submitData];
+  }
+
+  // Compute signature once.
+  // Not computing this beforehand because it might be a very computational expensive operation
```

Comment:
Haven't really checked but AFAIK lambdas run in a very small instance on AWS. The main idea was to prevent any issues with multiple requests being fulfilled simultaneously. Do you think we should pre-compute this signature for all requests during provider initialization no matter if the fulfill function is going to be called?